ecosystem/typescript/sdk/src/util.test.ts

```diff
@@ -1,5 +1,6 @@
 export const NODE_URL = process.env.APTOS_NODE_URL || "https://fullnode.devnet.aptoslabs.com";
 export const FAUCET_URL = process.env.APTOS_FAUCET_URL || "https://faucet.devnet.aptoslabs.com";
+export const CHAIN_ID = process.env.APTOS_CHAIN_ID || "14";
```

Comment:
you should be able to get chain id from the chain, would probably recommend that for now, see the python sdk: https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/python/sdk/aptos_sdk/client.py#L32

Reply:
Sounds good.